recipe/meta.yaml

```diff
@@ -57,7 +57,7 @@ requirements:
     - numpy
     - pandas >=1.0.0,<2.0.0
     - protobuf >=3.6.1,<4
-    - pyarrow >=4.0.0,<7.0.0
```

Comment:
Please update (all?) of the reqs with the updated pins in

https://github.com/flyteorg/flytekit/blob/master/setup.py

Reply:
Gotcha, done